### ISSUE TITLE
replace orcid and orcid_platform by NXidentifier and allow for multiple also for em, correct immediate subgroup inside a coordinate system set

### ISSUE THREAD

**Issue by mkuehbach** [FAIRmat-NFDI/data-modeling#145]

**Comment by domna** [comment]
@mkuehbach Could you link the branch/PR which you showed in the DSM today here?
I think it would also be nice to have NXidentifier available in the fairmat branch so we already could start using it in the NXmpes refactoring.

**Comment by domna** [comment]
https://github.com/FAIRmat-NFDI/nexus_definitions/pull/51

**Comment by lukaspie** [comment]
Closed as this will be accomplished by https://github.com/nexusformat/definitions/pull/1486 once the new NIAC release is done.
